script.js

```diff
@@ -22,16 +22,16 @@ ulCont.addEventListener('click', ((e) => {
   }
 }));
 
-// const openModal = function () {
-//   overlay.classList.remove('hidden');
-//   modal.classList.remove('hidden');
-// };
+const openModal = function () {
+  overlay.classList.remove('hidden');
+  modal.classList.remove('hidden');
+};
 
-// openModalBtn.addEventListener('click', openModal);
+openModalBtn.addEventListener('click', openModal);
 
-// const closeModal = function () {
-//   modal.classList.add('hidden');
-//   overlay.classList.add('hidden');
-// };
+const closeModal = function () {
+  modal.classList.add('hidden');
+  overlay.classList.add('hidden');
+};
 
-// closeModalBtn.addEventListener('click', closeModal);
+closeModalBtn.addEventListener('click', closeModal);
```

Comment:
- [X] I would suggest we stick to the project requirement as much as possible, currently, the `See project` button is not functioning properly for the modal pop-up, only the button in the first card is responding to the modal pop-up when clicked on while others are not. I would suggest you create it dynamically and `loop` through the array.